stable/concourse/README.md

```diff
@@ -162,58 +162,43 @@ $ helm install --name my-release -f values.yaml stable/concourse
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
-### SSH Keys
+### Secrets
 
-To run Concourse securely you'll need [3 private keys](https://concourse.ci/binaries.html#generating-keys). For your convenience, this chart provides some [default keys](concourse-keys), but it is recommended that you generate your own keys by running:
+For your convenience, this chart provides some default values for secrets, but it is recommended that you generate and manage these secrets outside the Helm chart. To do this, set `secrets.create` to `false`, create files for each secret value, and turn it all into a k8s secret. First, perform the following to create the manditory secret values:
```

Comment:
@viglesiasce any idea how we run automated testing on this if mandatory secret values need to be manually created?

It would be nice if you could pass in a secret to use that was created outside the chart. If that didn't happen the chart could generate a secret.